tailscale/config.yaml

```diff
@@ -19,6 +19,7 @@ arch:
 init: false
 hassio_api: true
 host_network: true
+host_dbus: true
```

Comment:
This is a very unwanted addition for add-ons. We should remove the dependency on the dbus.

Reply:
I see... we need this.